helpers/pyband/pyband/client.py

```diff
@@ -30,8 +30,10 @@ def _get(self, path, **kwargs):
     def _get_result(self, path, **kwargs):
         return self._get(path, **kwargs)["result"]
 
-    def send_tx(self, data: dict) -> dict:
-        return requests.post(self.rpc_url + "/txs", json=data).json()
+    def send_tx(self, data: dict, mode="sync") -> dict:
```

Comment:
should return type be dict?

Reply:
I'll make another PR for Client class spec.